fdbserver/masterserver.actor.cpp

```diff
@@ -466,12 +493,21 @@ ACTOR Future<Void> updateLogsValue( Reference<MasterData> self, Database cx ) {
 	}
 }
 
-Future<Void> sendMasterRegistration( MasterData* self, LogSystemConfig const& logSystemConfig, vector<MasterProxyInterface> proxies, vector<ResolverInterface> resolvers, DBRecoveryCount recoveryCount, vector<UID> priorCommittedLogServers ) {
+Future<Void> sendMasterRegistration(
+    MasterData* self,
+    LogSystemConfig const& logSystemConfig,
+	vector<MasterProxyInterface> proxies,
+	vector<GrvProxyInterface> grvProxies,
+	vector<ResolverInterface> resolvers,
+	DBRecoveryCount recoveryCount,
+	vector<UID> priorCommittedLogServers ) {
```

Comment:
I see mixed tab/space used here. Install a [clang-format-hooks](https://github.com/barisione/clang-format-hooks) can do this automatically for you before commit.